packages/ui-extensions/src/surfaces/customer-account/components/ResourceItem/ResourceItem.ts

```diff
@@ -0,0 +1,42 @@
+import {createRemoteComponent} from '@remote-ui/core';
+import type {RemoteFragment} from '@remote-ui/core';
+import {LinkProps} from '../shared-checkout-components';
+
+export interface ResourceItemProps
+  extends Pick<LinkProps, 'accessibilityLabel' | 'to' | 'onPress'> {
+  /**
+   * A label that describes the purpose or contents of the item. When set,
+   * it will be announced to buyers using assistive technologies and will
+   * provide them with more context.
+   */
+  accessibilityLabel?: string;
```

Comment:
You picked `accessibilityLabel` and specified it here. You only need it once

Reply:
Thanks, fixed!